### ISSUE TITLE
480i not working

### ISSUE THREAD

**Issue by cathoderaydude** [psakhis/Groovy_MiSTer#2]
Any attempt to display 480i (PSX / Saturn games, etc.) seems to produce 240p. Testing with Retroarch.

When Saturn games switch into 480 mode (e.g. Die Hard Arcade) the Mister OSD reports double the horizontal res, but still only 240 on the vertical, and scanlines are clearly visible on the CRT.

When PSX goes to 480 (e.g. boot logo, Chrono Cross menu) it appears to be splitting the lines of the 480i image into 240p, so we get an odd line, then an empty line, then an even line, then an empty line, and so on.

Is this expected behavior? I kinda figured 240p might be the only supported mode since it's what 95% of stuff uses and this is really early in development, but I wanted to make sure that was true.

By the way, this is the coolest project I've ever seen. I've been hoping someone would do this for literally more than 20 years, and wished I had the skill to do it myself. Thank you so much, I'm excited to see future developments.

**Comment by psakhis** [comment]
I will check, only tested interlaced with MAME and Mednafen for PSX and Saturn.
I tested x68000 and dosbox for retroarch on interlaced but not PSX or Saturn cores.

**Comment by cathoderaydude** [comment]
So a little more info - I switched to Mednafen and it _looks_ like it's working, but I don't believe it actually is.

When Chrono Cross (for instance) enters 480i for menus, the MiSTer OSD reports that it's entering a 480 mode, and I can tell both pairs of lines are being drawn on the CRT, but the picture still looks like it's missing all the even lines.

I took a slowmo video with my phone, and what I believe is going on is that it's drawing the odd field, then repeating it again for the even field. Let me know if more info about my setup or pictures would be helpful.
